app/components/Typography.tsx

```diff
@@ -12,13 +12,15 @@ interface TypographyProps {
   style?: TextStyle;
   testID?: string;
   children: JSX.Element | string;
+  onPress?: () => void;
```

Comment:
What do you think about not adding this prop and using a touchable opacity at the component itself?  I worry about this component becoming gradually a kitchen drawer of a maintainability headache.

Additionally it would be really great to fully implement RTL language layout eventually and keeping this super lean will help with that.

Reply:
The behavior is improved with using the direct onPress IMO -- it delegates to the native text link behavior, which gives us immediate wins on accessibility and this way the topographies are better set up to be composed as Text elements.

Maybe we can have it directly extend a Text component instead, from a props perspective?

I'm unfamiliar with RTL language layout -- what would that entail?